examples/testapp/src/context/ConfigContextProvider.tsx

```diff
@@ -107,6 +119,13 @@ export const ConfigContextProvider = ({ children }: ConfigContextProviderProps)
     setScwUrl(url);
   }, []);
 
+  const setSubAccountsConfig = useCallback(
+    (subAccountsConfig: SubAccountOptions) => {
+      setSAConfig((prev) => ({ ...prev, ...subAccountsConfig }));
+    },
+    []
+  );
+
```

Comment:
@stephancill updated the naming to support this change https://github.com/coinbase/coinbase-wallet-sdk/pull/1606/files/323a8dd16555ad2adac51056c16f675f32b24ecc#r2056863896

Reply:
ahhh okay i see